docs/functions/function-crd.md

```diff
@@ -333,6 +333,48 @@ spec:
 
 For more information about probe types, probe check mechanisms, and probe parameters, see Kubernetes documentation on [Pod lifecycle](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle) and [configure probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes).
 
+## Security context
+
+A security context defines privilege and access control settings for a Pod. By default, Function Mesh uses the following `PodSecurityContext` as the default value and applies to every function's Pod.
```

Comment:
I think we need to apply these changes to other CRDs like sink and source as well.